validator/client/validator.go

```diff
@@ -1075,7 +1068,20 @@ func (v *validator) buildProposerSettingsRequests(ctx context.Context, pubkeys [
 			signedRegisterValidatorRequests = append(signedRegisterValidatorRequests, request)
 		}
 	}
-	return validatorToFeeRecipients, signedRegisterValidatorRequests, nil
+
+	if len(proposerFeeRecipientRequests) != len(pubkeys) {
+		log.WithFields(logrus.Fields{
+			"activePubkeys": len(pubkeys) - len(proposerFeeRecipientRequests),
```

Comment:
`activePubkeys` is less obvious to me. How about have these `totalRequested` and `totalRegistered`?